### DISCUSSION TITLE
Dynamic label parameter

### DISCUSSION THREAD

**Discussion by andrew-tud** [SAP/luigi#3021] [Q&A] [answered]
Hi, 

I am trying to have a dynamic pathSegment and have it displayed as the label as per this config:
```
{
      category: {
        label: 'iFrame 2 c',
        icon: 'home',
        collapsible: true
      }, 
      pathSegment: ':dynamic',
      label: ':dynamic',
      viewUrl: 'http://127.0.0.1:8080/:dynamic',
      hideFromNav: false,
      loadingIndicator: {
        enabled: false
      },
    },
```

but the label or the url link is displaying ":dynamic" on the web page's side navigation while the viewUrl and pathsegment is working fine. Is there a way to reflect the dynamic parameter to the label as well? And is it possible to do some string manipulation for the dynamic label to make it more user friendly?

There might be a workaround for this by using my own navigation panel in a vertical split architecture. But I am wondering if this can be done in the side navigation of the MFE Shell as a horizontal split architecture.

**Comment by ndricimrr** [comment, marked as answer]
Hi @andrew-tud ,
Please follow the answer [from here](https://github.com/SAP/luigi/discussions/3023#discussioncomment-4314753) as it seems to respond to this question as well
